tensorflow/tsl/util/BUILD

```diff
@@ -26,9 +26,19 @@ package(
     licenses = ["notice"],
 )
 
+filegroup(
+    name = "mobile_srcs_byteswaparray",
+    srcs = [
+        "byte_swap_array.cc",
+        "byte_swap_array.h",
+    ],
+)
+
 filegroup(
     name = "mobile_srcs_only_runtime",
     srcs = [
+        "byte_swap_array.cc",
+        "byte_swap_array.h",
```

Comment:
Remove these from here since they are needed for `mobile_srcs_no_runtime`, and including them in both leads to duplicate symbol errors.

Reply:
Removed.